client/app/pages/dashboards/DashboardPage.jsx

```diff
@@ -144,7 +147,7 @@ function DashboardComponent(props) {
         </div>
       )}
       {editingLayout && <DashboardSettings dashboardConfiguration={dashboardConfiguration} />}
-      <div id="dashboard-container">
+      <div id={dashboardContainerId}>
```

Comment:
This one is different - it's used for styling:
https://github.com/getredash/redash/blob/d8d7c78992e44a4b6d7fdd4c39ccc1c35cd8c7a9/client/app/pages/dashboards/DashboardPage.less#L16-L19